activerecord/lib/active_record/relation.rb

```diff
@@ -573,8 +574,40 @@ def touch_all(*names, time: nil)
     # ==== Examples
     #
     #   Person.where(age: 0..18).destroy_all
-    def destroy_all
-      records.each(&:destroy).tap { reset }
+    #
+    # If +config.active_record.destroy_all_in_batches+ is set to +true+, it will ensure
+    # to perform the record's deletion in batches
+    # and destroy_all won't longer return the collection of the deleted records
+    #
+    # ==== Options
+    # * <tt>:start</tt> - Specifies the primary key value to start from, inclusive of the value.
+    # * <tt>:finish</tt> - Specifies the primary key value to end at, inclusive of the value.
+    # * <tt>:batch_size</tt> - Specifies the size of the batch. Defaults to 1000.
+    # * <tt>:error_on_ignore</tt> - Overrides the application config to specify if an error should be raised when
+    #   an order is present in the relation.
+    # * <tt>:order</tt> - Specifies the primary key order (can be :asc or :desc). Defaults to :asc.
+    #
+    # NOTE: These arguments are honoured only if +config.active_record.destroy_all_in_batches+ is set to +true+
```

Comment:
Period is missing, I guess?

```suggestion
    # NOTE: These arguments are honoured only if +config.active_record.destroy_all_in_batches+ is set to +true+.
```